src/extensions/tos.ts

```diff
@@ -0,0 +1,120 @@
+import { Client } from "../apiv2";
+import { extensionsTOSOrigin } from "../api";
+import { logger } from "../logger";
+import { confirm } from "../prompt";
+import { FirebaseError } from "../error";
+
+const VERSION = "v1";
+const extensionsTosUrl = (tos: string) => `https://firebase.google.com/terms/extensions/${tos}`;
```

Comment:
Suggestion: have a constant for each -- IMO it's nice to be able to search constant strings if they need replacing, where they're used, etc.